src/operator/operator_tune-inl.h

```diff
@@ -56,7 +56,7 @@ namespace op {
 #endif
 #endif  // MXNET_NO_INLINE
 
-#define OUTSIDE_COUNT_SHIFT    9
+#define OUTSIDE_COUNT_SHIFT  3
```

Comment:
Could you elaborate on this? Maybe add a comment in the code why you made that change

Reply:
The tuning code is trying to run a trivial code(int x+=y) and then average the timing stats over all the runs. 2^OUTSIDE_COUNT_SHIFT is one factor in determining number of iterations.
if OUTSIDE_COUNT_SHIFT 9 , it will run the trivial code these many time:
for 2^outside_count_shift
  for wl_count times, see here: // https://github.com/apache/incubator-mxnet/blob/master/src/operator/operator_tune-inl.h#L339 
    run_trivial_code

return total_time_taken_to_run_all_iteration / 2^outside_count_shift

Earlier the outside loop was running for 2^9 times, not it will be running for 2^3 times and averaging after that. It's just one way to reduce total number of iterations used to determine operator tuning.

Reply:
Ah I see, thanks a lot! How big is the time difference in between the old and the new setting?

I would imagine that 2^9 greatly benefits from caching while 2^3 is too trivial to benefit of the Caches and thus might influence the measurement in a way which doesn't reflect the reality (I assume we run a lot on caches).

Reply:
"How big is the time difference in between the old and the new setting?"
On c5.18xl with 72 cores, launching 30 mxnet processes,
It almost never finishes, I gave up after 10 minutes, some benchmark says it took them 1500 seconds. 
With new change it takes ~41 seconds to launch 30 processes.

Reply:
And if you limit the number of used cores using the environment variable?
Basically a comparison of all combinations of limiting the cpu cores and the iterations